src/inputs.c

```diff
@@ -89,28 +89,24 @@
 #define INPNAME_SWH0(x,y)      x = _tr_noop("SW H"); y = 0
 #define INPNAME_SWH1(x,y)      x = _tr_noop("SW H"); y = 1
 
-#define SWITCH_NAME_GEAR0 _tr("GEAR")
-#define SWITCH_NAME_GEAR1 _tr("GEAR")
-#define SWITCH_NAME_GEAR0 _tr("GEAR")
-
```

Comment:
Why was GEAR removed?

Reply:
not used anywhere

Reply:
I tracked the history, it is added by you but it is never used.